components/backup-stream/src/subscription_manager.rs

```diff
@@ -43,10 +41,72 @@ use crate::{
     Task,
 };
 
-type ScanPool = yatp::ThreadPool<yatp::task::callback::TaskCell>;
+type ScanPool = tokio::runtime::Runtime;
+
+thread_local! {
+    static THREAD_LOCAL_CDC_HANDLE: RefCell<Option<Box<dyn std::any::Any>>> = RefCell::default();
```

Comment:
Could you add some comments about why it has to be TLS?

Reply:
TBH TLS is the initial design(copied from how `coprocessor` handles the `!Sync` raftstore router, which also takes good from local reading), but for now I think perhaps adding a `Rwlock` to it will be good enough(we are just sending messages and won't really use ) and simpler, how do you think?

Reply:
Yes, it's clear and simpler.